ec2/sc-ec2-linux-jumpcloud-notebook-v1.0.0.yaml

```diff
@@ -22,7 +22,7 @@ Metadata:
 Mappings:
   NotebookTypes:
     Rstudio:
-      AMIID: "ami-01866cea595e4f968"  # https://github.com/Sage-Bionetworks-IT/packer-rstudio/tree/v1.0.3
+      AMIID: "ami-022e9a10f00f8ab9a"  # https://github.com/Sage-Bionetworks-IT/packer-rstudio/tree/v1.0.5
```

Comment:
For tag v1.0.5, this should be ami-0398809a941f2fa22, according to the list of AMIs in the EC2 console.

Reply:
thanks, fixed